openapi/components/paths/users.yaml

```diff
@@ -194,6 +194,85 @@ paths:
           $ref: ../responses/MissingCredentialsError.yaml
       security:
         - authCookie: []
+  '/users/{userId}/feedback':
```

Comment:
```suggestion
  '/users/{userId}/feedback':
    depricated: true
```

Reply:
spec liter complains

Reply:
you are right. was on the wrong path. Added another comment